tutorials/napps/publishing_your_napp.rst

```diff
@@ -153,7 +146,7 @@ README.rst
 ==========
 
 Among other things, the **README.rst** will be presented as the main content of
-the NApp page on the NApps Server (https://napps.kytos.io/<username>/<name>).
+the NApp page on |github|_  (https://github.com/<username>/<nappName>).
```

Comment:
Maybe we can keep the `<name>` to reflect the same term that the kytos-util cli uses and to be aligned with the previous paragraph too